backend/ohq/views.py

```diff
@@ -551,6 +554,24 @@ def post(self, request, course_pk, format=None):
         )
 
 
+class QueueStatisticViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
+    """
+    list:
+    Return a list of statistics - multiple data points for list statistics and heatmap statistics
+    and singleton for card statistics.
+    """
+
+    serializer_class = QueueStatisticSerializer
+    filter_backends = [DjangoFilterBackend]
+    filterset_class = QueueStatisticFilter
+    permission_classes = [QueueStatisticPermission | IsSuperuser]
+
+    def get_queryset(self):
+        # might need to change qs if students shouldn't be able to see all queue statistics
```

Comment:
I think it's fine for queue statistics to be visible for students, since they're very macro averages and does not contain sensitive info. (Although I forgot that we haven't included analytics for the student page yet - we will get to this in another PR @conswang )